Enterprise-CMCS/macpro-mdct-mfp
# Funding Source Option, Specify Bugfix

### Description
Fixed bug so Option, specify text shows up on funding source cards.

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/fb60ca83-df7d-4468-bd7c-4acf3aaefbc4

### Related ticket(s)
MDCT-2984-[https://qmacbis.atlassian.net/browse/MDCT-2984]

---
### How to test
1. Go to funding source page
2. Add new funding source - select 'Other, Specify' and enter a custom funding source
3. Fill out rest of modal
4. The custom funding source should show up in the header of the card


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
